### ISSUE TITLE
Issue with test-openldap in Travis-CI integration tests

### ISSUE THREAD

**Issue by eblondel** [rroemhild/docker-test-openldap#11]
Hello, thanks for this docker image. I'm now using it to test a draft R package LDAP client [here](https://github.com/eblondel/ldap4R). On local windows machine, the few integration tests i've run work, but on Travis-CI (see https://travis-ci.org/eblondel/ldap4R), on linux OS, the LDAP request to list of person objects:

**ldap://localhost:389/dc=planetexpress,dc=com?uid,cn,sn,givenName,mail,title,ou,employeeType,objectClass?sub?(objectClass=person)**

retrieves only the first person:

```
DN: cn=Amy Wong+sn=Kroker,ou=people,dc=planetexpress,dc=com
	objectClass: top
	objectClass: person
	objectClass: organizationalPerson
	objectClass: inetOrgPerson
	cn: Amy Wong
	sn: Kroker
	givenName: Amy
	mail: amy@planetexpress.com
	ou: Intern
	uid: amy
```
instead of the 7 persons listed in the test ldap.

Any idea what is wrong?

**Comment by rroemhild** [comment]
Hi sorry no idea. It could be a problem with R on linux?

**Comment by eblondel** [comment]
i don't think it comes from R. As you can see above the result of the LDAP is wrong, it returns one item instead of all. If there is an issue it's more likely to be with openldap on linux.
